src/server/bg_services/objects_reclaimer.js

```diff
@@ -42,6 +43,9 @@ class ObjectsReclaimer {
         if (has_errors) {
             return config.OBJECT_RECLAIMER_ERROR_DELAY;
         }
+
+        await clean_md_store_objects(Date.now());
```

Comment:
don't think this belongs to here... why should we run db_cleaner inside object_reclaimer - db_cleaner will run on objects with no regard to the list that was reclaimed (and of course we don't want to db-delete objects that was just deleted the DB!). I think what we wanted here is a new functionality that for each of the objects will check how many deleted objects we have with the same key and if it's more than X - to delete the older copies - @dannyzaken to keep me honest here...

Reply:
The jira issue (https://issues.redhat.com/browse/DFBUGS-1339) is specifically mentioned that we can call clean the deleted objects from the md store in object reclaimer. Thought this is the place. @dannyzaken Please correct me here

Reply:
@vh05 I didn't intend for the description in Jira to be specific instructions on how to fix it. I wanted to provide some general context. 
I don't see a reason to call from object_reclaimer to the db_cleaner. These are two separate bg_workers.